### DISCUSSION TITLE
What the plan about go-agent to add the kafka reporter

### DISCUSSION THREAD

**Discussion by xiaozaiyuji** [apache/skywalking#11507] [Q&A]
I use the skywalking as the go agent and want to know when the kafka reporter will ready.I was so excited to listern for this.

**Comment by mrproliu** [comment]
There is currently no plan to implement the Kafka reporter. If you are interested, you can participate in its development.

**Reply by wu-sheng** [reply]
Agreed. Usually, advanced reporter could be considered after most 1.0 features are ready.

**Reply by wu-sheng** [reply]
@xiaozaiyuji We hope to keep the kernel as good as possible, and widely tested, first.
